composer.json

```diff
@@ -65,25 +65,29 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.4",
         "yiisoft/arrays": "^3.0@dev",
-        "yiisoft/factory": "^3.0@dev",
         "yiisoft/strings": "^3.0@dev",
         "yiisoft/view": "^3.0@dev",
+        "yiisoft/factory": "^3.0@dev",
         "yiisoft/data": "^3.0@dev",
-        "php": "^7.4"
+        "ext-json": "*"
     },
     "require-dev": {
         "yiisoft/di": "^3.0@dev",
+        "yiisoft/event-dispatcher": "^3.0@dev",
         "yiisoft/log": "^3.0@dev",
```

Comment:
Doesn't seem to be used in tests.

Reply:
It's used in `config/tests.php` in the configuration for `View`